.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
           - --max-line-length=99
 
   - repo: https://github.com/PyCQA/isort
-    rev: "5.10.1"
+    rev: "5.12.0"
```

Comment:
could we fix this in a separate MR, or rebase if it already happened?

Reply:
Rebased, and it's gone now.